docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,534 @@
+  labels:
+    eks.amazonaws.com/component: coredns
+    k8s-app: kube-dns
+  name: coredns
+  namespace: kube-system
+data:
+  Corefile: |
+    .:53 {
+        errors
+        health {
+            lameduck 5s
+          }
+        ready
+        kubernetes cluster.local in-addr.arpa ip6.arpa {
+          pods insecure
+          fallthrough in-addr.arpa ip6.arpa
+        }
+        prometheus :9153
+        forward . /etc/resolv.conf
+        cache 30
+        loop
+        reload
+        loadbalance
+    }
+    camunda-paris.svc.cluster.local:53 {
+        errors
+        cache 30
+        forward . 10.202.30.193 10.202.46.72 10.202.65.75 {
+            force_tcp
+        }
+    }
+    camunda-paris-failover.svc.cluster.local:53 {
+        errors
+        cache 30
+        forward . 10.202.30.193 10.202.46.72 10.202.65.75 {
+            force_tcp
+        }
+    }
+```
+
+  </summary>
+</details>
+
+2. Repeat **Step 1** for the Paris cluster to configure the CoreDNS of Paris to redirect any DNS requests to the London DNS resolver for your chosen namespace, like `camunda-london`.
+
+3. Check that CoreDNS has reloaded for the changes to take effect before continuing. Make sure it contains `Reloading complete`:
+
+```bash
+kubectl --context $CLUSTER_0 logs -f deployment/coredns -n kube-system
+kubectl --context $CLUSTER_1 logs -f deployment/coredns -n kube-system
+```
+
+</TabItem>
+<TabItem value="test-dns-chaining" label="(optional) Test DNS chaining">
+
+A setup to test that the DNS chaining is working by using nginx pods and services to ping each other.
+
+1. Deploy the following nginx pod and service to each cluster within the `camunda-london` and `camunda-paris` namespaces:
+
+```bash
+kubectl --context $CLUSTER_0 apply -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_0
+kubectl --context $CLUSTER_1 apply -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_1
+```
+
+2. Try to curl each other from within the other pod:
+
+```bash
+kubectl --context $CLUSTER_0 exec -n $CAMUNDA_NAMESPACE_0 -it sample-nginx -- curl http://sample-nginx.sample-nginx-peer.$CAMUNDA_NAMESPACE_0.svc.cluster.local
+
+kubectl --context $CLUSTER_1 exec -n $CAMUNDA_NAMESPACE_1 -it sample-nginx -- curl http://sample-nginx.sample-nginx-peer.$CAMUNDA_NAMESPACE_1.svc.cluster.local
+```
+
+3. After a successful retrieval of the remote nginx start page, you can remove this temporary setup.
+
+```bash
+kubectl --context $CLUSTER_0 delete -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_0
+kubectl --context $CLUSTER_1 delete -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_1
+```
+
+</TabItem>
+</Tabs>
+
+## Deploy Camunda 8 to the clusters
+
+<Tabs queryString="deploy">
```

Comment:
Same as for previous tabs incarnation, it's not obvious that a user needs to go through steps one by one.
I suggest:
- adding `Step <N>. ` prefix to every tab name
- at the end of every tab content, add an explicit text to go to the next tab `Step <N+1>. `

Reply:
I'm aware of this, I wanted to first explore whether we keep the tabs or not, now that the general procedure is shorter due to the switch to cloning the repo. As now we have a part without tabs and the rest with tabs.

Nevertheless, good suggestion with the step if we keep the tabs.